addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -1439,6 +1439,51 @@ export function makeActionManager(env, router = _router) {
             }
             throw new InvalidButtonParamsError("Missing type for doActionButton request");
         }
+        if (action.embedded_action_ids && !this.currentController?.config.currentEmbeddedActionId) {
```

Comment:
in what kind of flows is the second part of the condition necessary?

Reply:
If you've already selected an embedded action. Because from there, if you select the main embedded action of the top bar, it would go through this code again and load the first embedded action of the top bar again.

Reply:
mmmh ok, let's move on with this, I challenge it later if necessary